translator/translate/otel/common/eksdetector.go

```diff
@@ -29,8 +34,11 @@ const (
 var _ Detector = (*EksDetector)(nil)
 
 var (
-	detectorSingleton Detector
-	once              sync.Once
+	detector            Detector
+	isEKSCacheSingleton IsEKSCache
+	once                sync.Once
+
+	EKSTrue bool
```

Comment:
is this used anywhere ? if not it can be removed

Reply:
nope, will remove